AzureMonitorAgent/agent.py

```diff
@@ -956,6 +957,15 @@ def parse_context(operation):
             raise ParameterMissingException
     return hutil
 
+def set_os_arch():
+    """
+    Checks if the system is x86 or aarch64 based and replaces package name 
```

Comment:
Can we make this more generic to allow for other architectures?

Reply:
We could have a list of supported architecture as a global variable and modify set_os_arch() as follows- 

supported_arch = ["x86_64","aarch64"]

def set_os_arch():

    global BundleFileName, supported_arch
    current_arch = platform.machine()
    
    for arch in supported_arch:
        if arch == current_arch:
            BundleFileName = BundleFileName.replace('x86_64',arch)